docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -0,0 +1,211 @@
+
+2.  Identify the current version of the `apk-tools-static` package.  You will need to navigate into the `main/x86_64` directory of your chosen release.  For example, the latest stable would be (http://nl.alpinelinux.org/alpine/latest-stable/main/x86_64/)[http://nl.alpinelinux.org/alpine/latest-stable/main/x86_64/].  From there, simply search for `apk-tools-static`.  Once you have found it, you will need to copy the file's location.  (Right click, Copy Link Address)
+
+3.  Download and extract the `apk-tools-static` package.  You should still be in `/alpine`
+        curl -s `apk-tools-static.apk` | tar xz
+
+4.  Perform initial distro install.  This will preform the install using the latest stable build of Alpine.
+        ./sbin/apk.static --repository http://nl.alpinelinux.org/alpine/latest-stable/main/ --update-cache --allow-untrusted --root /alpine --initdb add alpine-base alpine-mirrors
+
+### Initial setup
+
+1.  Configure your fstab.  Note, you need a single hard tab between each column.
+{: .file }
+/alpine/etc/fstab
+:   ~~~ conf
+    /dev/sdb    /   ext4    defaults,noatime    0   0
+    /dev/sda    /boot   ext4    defaults,noatime    0  1
+    /dev/sdc    swap    swap    defaults    0   0
+    ~~~
+
+2.  Configure the inittab
+{: .file }
+/alpine/etc/inittab
+:   ~~~ conf
+    # /etc/inittab
+    
+    ::sysinit:/sbin/rc sysinit
+    ::sysinit:/sbin/rc boot
+    ::wait:/sbin/rc default
+    
+    # Put a getty on the serial port
+    ttyS0::respawn:/sbin/getty -L ttyS0 115200 vt100
+    
+    # Stuff to do for the 3-finger salute
+    ::ctrlaltdel:/sbin/reboot
+    
+    # Stuff to do before rebooting
+    ::shutdown:/sbin/rc shutdown
+    ~~~
+
+3.  Create the Grub 2 boot config
+        mkdir /alpine/boot/grub
+{: .file }
+/alpine/boot/grub/grub.cfg
+:   ~~~ conf
+    set root=(hd0)
+    set default="Alpine Linux"
+    set timeout=0
+    
+    menuentry "Alpine Linux" {
+              linux /vmlinuz-grsec root=/dev/sdb modules=sd-mod,usb-storage,ext4 console=ttyS0 quiet
+              initrd /initramfs-grsec
+    }
+    ~~~
+    
+4.  Configure mkinitfs
+        mkdir /alpine/etc/mkinitfs
+{: .file }
+/alpine/etc/mkinitfs/mkinitfs.conf
+:   ~~~ conf
+    features="ata ide scsi virtio base ext4"
+    ~~~
+
+5.  Copy resolv.conf in
+        cp /etc/resolv.conf /alpine/etc
+        
+6.  Add `ttyS0` to securetty to allow root login over Lish
+        echo ttyS0 >> /alpine/etc/securetty
+
+### Installing the kernel and creating the initramfs
+
+1.  Bind proc and dev into the chroot
+        mount --bind /proc /alpine/proc
+        mount --bind /dev /alpine/dev
+
+2.  Enter the chroot
+        chroot /alpine /bin/sh
+
+3.  Select a mirror.  You can use the `f` option to automatically pick the fastest mirror.
+        setup-apkrepos
+
+4.  Update packages
+        apk update
+
+5.  Setup your hostname.
+        setup-hostname -n `hostname`
```

Comment:
if you don't call out that they should replace "hostname" with an actual hostname, you'll end up with a fleet of alpine users whose hostnames are literally that.